alerts_back_out.sh

```diff
@@ -0,0 +1,45 @@
+#! /bin/sh
+# Run on r2d2-v because dump monitor and acorn scripts are on /data/mta
+# Run as user mta
+
+git_dir=/home/malgosia/git/alerts_update
+
+# Space weather
+swdir=/data/mta4/space_weather
+
+cd $git_dir
+git checkout master
+
+cp -f $git_dir/space_weather/aceviolation_protons.csh $swdir/
+cp -f $git_dir/space_weather/ace_12h_viol.pl $swdir/
+cp -f $git_dir/space_weather/G13_yellow_viol.pl $swdir/
+cp -f $git_dir/space_weather/goes_hrc_proxy_viol.pl $swdir/
+cp -f $git_dir/space_weather/G13_red_viol.pl $swdir/
+cp -f $git_dir/space_weather/ace_invalid_spec.csh $swdir/
+cp -f $git_dir/space_weather/aceviolation_protonsP3_P5.csh $swdir/
+cp -f $git_dir/space_weather/aceviolation_protonsP6.csh $swdir/
+cp -f $git_dir/space_weather/G15_yellow_viol.pl $swdir/
+cp -f $git_dir/space_weather/G15_red_viol.pl $swdir/
```

Comment:
This looks great, but I see there are other files in /data/mta4/space_weather with email addresses.  Is there another reference file that helps to see which ones are used (to confirm your list here is complete)?